docs/README_dfki_mcqdev.md

```diff
@@ -0,0 +1,20 @@
+We were contacted by DFKI to create an emission tracking app for Android and iOS that
+While first also considering development of a new code base, we finally stayed on the
+path of contributing to the existing repository because it wouldn't be a big difference
+in time needed for development but we could contribute to a greater project and maybe
+also profit from future community updates.
+
+The project is based on Cordova with Ionic, though it uses very old version and also
+lots of deprecated npm packages which makes a lot of newer Ionic features unavailable.
+
+Documentation is located in e-mission-docs repository.
+Es gibt insgesamt 4 repositories, die für uns relevant sind:
```

Comment:
Maybe change this to english too?

Reply:
sorry, didn't notice that. Now is translated